File: result/model/User_Quiz_Attempt_Response.py
from dataclasses import dataclass
from typing import Dict

from model.Question import Questions
from model.user_attempts import Attempts
from model.user_response import UserResponse

@dataclass
class UserQuizAttemptResponse:

    question : Questions

    userResponse : UserResponse

    @classmethod
    def from_dict(cls, data: Dict[str, any]) -> "UserQuizAttemptResponse":
        
        question = Questions(**data.get('question'))

        quizAttempt = Attempts
        userResponse = UserResponse(**data.get('userResponse'))

        

File: userservice/src/main/java/com/project/userservice/dto/UserQuizAttemptResponse.java
package com.project.userservice.dto;

import com.project.userservice.model.UserResponse;

import lombok.Data;

@Data
public class UserQuizAttemptResponse 
{

    private Question question;

    private UserResponse userResponse;

    public UserQuizAttemptResponse(Question question, UserResponse userResponse) 
    {
        this.question = question;
        this.userResponse = userResponse;
    }

}
